index.html

```diff
@@ -27,8 +27,16 @@
               <li class="nav-link">Contact</li>
             </ul>
         </nav>
+        <nav class="portfolioM">
+          <p class="portfolioB"><i class="fa fa-times"></i></p>
+          <ul class="portfoliomenu">
+            <li class="menu"><a href="#hero" id="mbmu">portfolio</a></li>
+            <li class="menu"><a href="#myself" id="mbmu">About</a></li>
+            <li class="menu"><a href="#form" id="mbmu">contact</a></li>
+          </ul>
+          </nav>  
```

Comment:
- [ ] As suggested by the previous reviewer, Take a look at your menu at a width of 517px which is still under mobile. You will note that it is no longer like the Figma. 
**Figma**
![menu](https://github.com/Danny-imo/my-project/assets/99656687/59aa4222-73a4-4c9c-b082-9d8ead9a64b8)

**Your design**
![menu](https://github.com/Danny-imo/my-project/assets/99656687/a9297661-951d-40c3-b063-e91d201616f5)